examples/README.md

```diff
@@ -56,21 +118,22 @@ docker-compose -f docker-compose.cluster.yml up -d
 then run the example:
 
 ```
-ts-node --transpile-only create_table_local_cluster.ts
+ts-node --transpile-only create_table_on_premise_cluster.ts
 ```
 
-- for `create_table_cloud.ts`, Docker containers are not required, but you need to set some environment variables first:
+### ClickHouse Cloud examples
+
+- for `*_cloud.ts` examples, Docker containers are not required, but you need to set some environment variables first:
 
 ```sh
 export CLICKHOUSE_HOST=https://<your-clickhouse-cloud-hostname>:8443
 export CLICKHOUSE_PASSWORD=<your-clickhouse-cloud-password>
 ```
 
-You can obtain these credentials in the Cloud console.
-This example assumes that you do not add any users or databases
-to your Cloud instance, so it is `default` for both.
+You can obtain these credentials in the ClickHouse Cloud console.
```

Comment:
should we add a link to the docs or a screenshot with the "Show connection string"?

Reply:
There is a "Gather your connection details" section. https://clickhouse.com/docs/en/integrations/language-clients/javascript#gather-your-connection-details

Reply:
Added the link.